src/server/game/DataStores/DB2Structure.h

```diff
@@ -98,8 +98,8 @@ struct AreaTableEntry
     // Checks if zone activates pvp talents.
     bool ActivatesPvpTalents() const
     {
-        return (!IsSanctuary() && (Flags[0] & AREA_FLAG_ARENA) != 0)
-            || ID == 7618 || ID == 8320 || ID == 8304 || ID == 8151 || ID == 8078 || ID == 8051 || ID == 8324 || ID == 8168 || ID == 8328;
+        return (!IsSanctuary() && (Flags[0] & (AREA_FLAG_ARENA | AREA_FLAG_WINTERGRASP)) != 0)
+            || ID == 8320 || ID == 8304 || ID == 8078 || ID == 8051 || ID == 8324 || ID == 8328 || ID == 5095;
```

Comment:
QuestInfoID 113 causes it I think.
```
SELECT ID, QuestType, LogTitle FROM quest_template WHERE QuestInfoID=113;
+-------+-----------+---------------------------------------------+
| ID    | QuestType | LogTitle                                    |
+-------+-----------+---------------------------------------------+
| 37295 |         3 | Kill Rylai Crestfall                        |
| 37297 |         3 | Kill Jeron Emberfall                        |
| 37299 |         3 | Bonus Objective: Kill Stormshield Priests   |
| 37475 |         3 | Molten Quarry                               |
| 37476 |         3 | Ashmaul Burial Grounds                      |
| 37477 |         3 | Brute's Rise                                |
| 37478 |         3 | Amphitheater of Annihilation                |
| 37479 |         2 | Bonus Objective: Kill Alliance Leaders      |
| 37480 |         2 | Bonus Objective: Kill Alliance Leaders      |
| 43601 |         3 | Warden Tower Assault: Whisperwind's Citadel |
| 43600 |         3 | Warden Tower Assault: Whisperwind's Citadel |
| 43599 |         3 | Warden Tower Assault: Blackhawk's Bulwark   |
| 43598 |         3 | Warden Tower Assault: Blackhawk's Bulwark   |
| 43303 |         2 | Time to Rumble!                             |
| 43248 |         3 | Warden Tower Assault: Darkfollow's Spire    |
| 43247 |         3 | Warden Tower Assault: Darkfollow's Spire    |
| 43183 |         3 | Warden Tower Assault: Starstalker's Point   |
| 42070 |         3 | Warden Tower Assault: Starstalker's Point   |
| 42023 |         3 | Black Rook Rumble                           |
| 41421 |         3 | Warden Tower Assault: Nightwatcher's Perch  |
| 41420 |         3 | Warden Tower Assault: Nightwatcher's Perch  |
| 41257 |         3 | Warden Tower Assault: Cordana's Apex        |
| 41227 |         3 | Warden Tower Assault: Cordana's Apex        |
| 41013 |         3 | Darkbrul Arena                              |
| 42025 |         3 | Bareback Brawl                              |
| 41896 |         3 | Operation Murloc Freedom                    |
| 45740 |         3 | Silithyst                                   |
| 45657 |         3 | Silithyst                                   |
+-------+-----------+---------------------------------------------+
```

Will remove it all those warden tower areas